packages/kbn-es-archiver/src/actions/rebuild_all.ts

```diff
@@ -20,11 +35,319 @@ import {
   createFormatArchiveStreams,
 } from '../lib';
 
+function makeid(length: number) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+export function createChrisStream<T>(fn: (value: T, i: number) => any[] | undefined) {
```

Comment:
I assume that all the changes in this file will be reverted before merging?

Reply:
Yes!